back-end/application.js

```diff
@@ -0,0 +1,59 @@
+// load .env data into process.env
+require("dotenv").config();
```

Comment:
This line will load the config for the `development` environment. I'm not sure that's what you want, in this file.

Reply:
Because this `application.js` file might be included from either `server.js` or by your tests, and you don't want to load the `development` config when running your tests.